src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1940,6 +1940,10 @@ public List<LibraryMirror> ResetVMAndImportLibrary(List<string> libraries)
         /// <returns></returns>
         public IDictionary<Guid, List<ProtoCore.Runtime.WarningEntry>> GetRuntimeWarnings()
         {
+            var ret = new Dictionary<Guid, List<ProtoCore.Runtime.WarningEntry>>();
+            if (runtimeCore == null)
```

Comment:
Sure. In the case of that defect, `UpdateGraphAsyncTask` completion callback actually comes in after the `EngineController` (i.e. including the `LiveRunner`) is disposed of. The callback tries to get warning/error messages from the `LiveRunner` but it is `null`.